### PR TITLE
LPS-169193

### PR DESCRIPTION
**Pull request by balazssk**
Hi Team,
This is a follow up of  https://github.com/liferay-lima/liferay-portal/pull/3766.
I've added Test cases

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/webserver/test/WebServerFriendlyURLTest.java [hunk 1]
```diff
@@ -69,6 +70,23 @@ public void testExistingFileEntryFriendlyURLHasFiles() throws Exception {
 		Assert.assertTrue(WebServerServlet.hasFiles(mockHttpServletRequest));
 	}
 
+	@Test
+	public void testExistingFileEntryFriendlyURLHasFilesParts2()
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/webserver/test/WebServerFriendlyURLTest.java:74]: Can you use proper names for the tests? e.g. `testGroupIdUUIDFriendlyURLHasFiles`.

File: portal-impl/src/com/liferay/portal/webserver/WebServerServlet.java [hunk 2]
```diff
@@ -1371,9 +1371,10 @@ private static void _checkFileEntry(String[] pathArray) throws Exception {
 			DLAppLocalServiceUtil.getFileEntry(fileShortcut.getToFileEntryId());
 		}
 		else if (pathArray.length == 2) {
+			long groupId = GetterUtil.getLong(pathArray[0]);
```
Comment (adolfopa) [inline portal-impl/src/com/liferay/portal/webserver/WebServerServlet.java:1374]: Can you inline this variable?

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/webserver/test/WebServerFriendlyURLTest.java (hunk 1 above)
Comment (balazssk) [reply modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/webserver/test/WebServerFriendlyURLTest.java]: cheers for the suggestion @adolfopa , implemented

File: portal-impl/src/com/liferay/portal/webserver/WebServerServlet.java (hunk 2 above)
Comment (balazssk) [reply portal-impl/src/com/liferay/portal/webserver/WebServerServlet.java]: roger

**Comment by balazssk** [comment]
ci:test:sf

**Comment by balazssk** [comment]
rebased to master to avoid conflict

**Comment by adolfopa** [comment]
ci:forward

**Comment by Hanlf** [comment]
ci:forward

**Comment by Hanlf** [comment]
The failure [com.liferay.headless.delivery.resource.v1_0.test.CommentResourceTest](https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/321446/testReport/com.liferay.headless.delivery.resource.v1_0.test/CommentResourceTest) only occurs on this PR, re-run it to check if it is falky issue.

**Comment by Hanlf** [comment]
ci:forward

**Comment by Hanlf** [comment]
The above failures are not related to this PR, sent it to BChan here: https://github.com/brianchandotcom/liferay-portal/pull/127039

**Pull request closed without merging**